machines/sns59.nix

```diff
@@ -26,6 +26,12 @@ in {
     fsType = "nfs4";
   };
 
+  # Expose ports for Faasten gateway
+  networking.firewall.allowedTCPPorts = [
+    # 8080 for requests, 1337 for RPCs
+    8080 1337
```

Comment:
So, this will allow access to those ports from anywhere on the internet. Instead, you probably want to constrain these rules to the particular interface:

```
networking.firewall.interfaces."enp7s0".allowedTCPPorts = [ 8080 1337 ];
```

Reply:
Fixed it